INSTALL.md

```diff
@@ -0,0 +1,84 @@
+# Walkthrough Instructions
+
+These instructions should walk you through using the dont script to setup a digital ocean droplet
+
+### Prerequisites
+
+Make sure you have your Digital Ocean api key available. Once you have that, copy `misc/dorc.example` to `~/.dorc` and paste your key on the api line in the config file. Otherwise, you'll have to supply the key every time you run the dont command (installed below).
+
+
+You'll need to have some commands available to start. Consult your local package manager if they aren't:
+
+1. python 2.7
+2. pip
+3. virtualenv
+4. gcc
+5. rsync
+
+The following development packages are needed as well:
+
+1. python
+2. openssl
+
+There are minimal instructions for centos 7 here that should get you going fairy quickly, execute these commands as root to bootstrap the machine quickly:
+```
+yum -y install http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum -y update
+yum -y install python-pip gcc python-devel openssl-devel rsync
+pip install virtualenv
+```
+
+Also, git mangles the permission on the ssh keys, correct them with: `chmod 0600 misc/hudsonalpha*`
+
```

Comment:
Be sure all files are in unix format by running the following command (not doing so causes errors when executing do/setup.py, among other files)

find /path/to/code -type f -exec dos2unix {} \;